README.md

```diff
@@ -124,26 +124,9 @@ The release process is as follows:
 
 ## Set logging level
 
-### Swift
+GoogleDataTransport will follow the logging level of the underlying GoogleUtilities logger. If GoogleDataTransport is being used with a Firebase App, you can enable debug logs by adding the command line argumen `-FIRDebugEnabled` to your app's Arguments Passed on Launch. You can disable debug logs with `-FIRDebugDisabled`.
```

Comment:
Two small nits:
1. Could you keep README lines to 80 char limit?
2. typo: argumen → argument